unit_tests/sources/declarative/partition_routers/test_async_job_partition_router.py

```diff
@@ -58,7 +58,7 @@ def test_stream_slices_with_parent_slicer():
         job_orchestrator_factory=lambda stream_slices: AsyncJobOrchestrator(
             MockAsyncJobRepository(),
             stream_slices,
-            JobTracker(_NO_LIMIT),
+            JobTracker(_NO_LIMIT, config={}),
```

Comment:
Seems like JobTracker already has config={} default in the initializer? Could avoid changes here I think?